tsconfig.json

```diff
@@ -13,5 +13,5 @@
   // NOTE: `test/**/*` is automatically included in `blockly-scripts start`.
   // Only src matters for production builds.
   "include": ["./src", "./test"],
-  "exclude": ["./test/webdriverio/"]
+  "exclude": ["./test/webdriverio/test"]
```

Comment:
Before this change, test/webdriverio/index.ts was covered by neither tsconfig.json file.

Reply:
I've pushed and rebased a simplified version of this change that only adds the new lint checks to the wdio tests to decouple it from the tsconfig.json discussion because on reflection I don't think it's that helpful for me to propose changes here.

I'd tried to do sometime simpler like this initially and somehow convinced myself that a bigger config change was needed but the different sections with different file globs combine just as you'd hope.